packages/build-info/src/frameworks/astro.test.ts

```diff
@@ -23,3 +23,30 @@ test.each([
   expect(detected?.[0].dev?.command).toBe('astro dev')
   expect(detected?.[0].dev?.port).toBe(3000)
```

Comment:
Looks like it's returning 3000 for non <3.0.0 ?

Reply:
Hmm yeah, I initially did it the inverse - default `3000` but then update for `>3.0.0`, but realised it's probably best to do the opposite as 4321 will now be the standard.

However, forgot to update the existing test